jobs/resource_job_test.go

```diff
@@ -1336,7 +1336,62 @@ func TestJobsAPIList(t *testing.T) {
 		a := NewJobsAPI(ctx, client)
 		l, err := a.List()
 		require.NoError(t, err)
-		assert.Len(t, l.Jobs, 1)
+		assert.Len(t, l, 1)
+	})
+}
+
+func TestJobsAPIListMultiplePages(t *testing.T) {
```

Comment:
what about duplicates that appear when job is added in the middle of iteration? :) you can skip this one here, though